examples/MultiPageDemo/MultiPageDemo.ino

```diff
@@ -0,0 +1,215 @@
+#include <Gui900.h>
+Gui900 gui(6, 9,3,1,1);
+#include <Fonts/FreeSerifBoldItalic24pt7b.h>    // Font: Serif, Bold, Italic
+unsigned long LastPress;
+byte R,G,B;
+int counter_count;
+void beep(int time=10){
+  #ifndef __SAM3X8E__
+    if(gui.checkbox_checked[5][0])
+        tone(48,1000,time);
+  #endif
+}
+void setup() {
+    // Start your hardware here.
+    Serial.begin(9600);
+    Serial.println("Start");
```

Comment:
I ran this example on Arduino Due and only received an 'S' instead of 'Start', The micro-controller will crash after 'S' is transmitted.